services/pkgreview/package-rating.service.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  name: Joi.string().required(),
+  type: Joi.string().required(),
+  icon: Joi.string().required(),
+  starString: [Joi.string().optional(), Joi.allow(null)],
+  reviewsCount: Joi.number(),
+  rating: [Joi.number().optional(), Joi.allow(null)],
+}).required()
```

Comment:
We use the schemas to ensure the data points that are required to render the badge are there (without having to check individual keys later on), so these don't need to mimic every single entry in the response; just the ones the service class needs.

As such, we can drop every key in the above schema that's not needed (that would be removing everything except `rating` and `reviewsCount`). Also some inline comments above on those two

Reply:
Ok, would do 🙌

**Question:** I am using `name` and `type` for the badge link, so they need to included in the schema, right?

Reply:
> so they need to included in the schema, right?

No, because the badge link needs to be removed